### PR TITLE
Declare this extension safe for parallel reading and writing

### LINKED ISSUE #16: Warning: "extension does not declare if it is safe for parallel reading"
When building with Sphinx 2.2.0, I get the following warning:
```
the sphinxcontrib.autoprogram extension does not declare if it is safe for parallel reading, assuming it isn't - please ask the extension author to check and make it explicit
```
Here's what a similar project did to fix this warning: https://github.com/agronholm/sphinx-autodoc-typehints/commit/c4657bc57a290a79605ed86398db28419aabfbb9

### PR DESCRIPTION
**Pull request by langston-barrett**
See https://www.sphinx-doc.org/en/master/extdev/#extension-metadata

Fixes #16

### CODE REVIEW

**Review by dahlia — APPROVED** [review]
:+1:

**Comment by dahlia** [comment]
There is already _docs/changes.rst_ file… do you have any plan to replace it with the new _changelog.md_ file?

**Comment by langston-barrett** [comment]
@dahlia Nope, I just didnt see that one! Thanks!

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+- Declare this extension safe for parallel reading
+
+## [0.1.5] - 2018-05-15
+
+[Unreleased]: https://github.com/sphinx-contrib/autoprogram/compare/v1.0.0...HEAD
+[0.1.5]: https://github.com/sphinx-contrib/autoprogram/compare/0.1.4...0.1.5
```
Comment (dahlia) [inline changelog.md:12]: Could you move them to *docs/changes.rst*?

**Review by dahlia — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (langston-barrett) [reply changelog.md:12]: Will do.

**Review by dahlia — APPROVED** [review]

**Comment by dahlia** [comment]
Some build jobs failed, but they seem due to an unrelated problem. (Travis CI does not support Python 2.6 and 3.3 anymore.) I am going to merge this.  Thanks!

**Pull request merged**